### PR TITLE
Creating all entities

### PR DESCRIPTION
**Pull request by khojianvar**

### CODE REVIEW

File: BioMed.Api/BioMed.Domain/Entities/Department.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace BioMed.Domain.Entities
+{
+    public class Department : EntityBase
+    {
+        public string? Name { get; set; }
+
+        public virtual ICollection<Spesialization> Spesializations { get;}
```
[the remark this answers is not in the document]
Comment (khojianvar) [reply BioMed.Api/BioMed.Domain/Entities/Department.cs]: Thank you for your explanation, i will correct my mistakes.

File: BioMed.Api/BioMed.Domain/Entities/Treatment.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿namespace BioMed.Domain.Entities
+{
+    public class Treatment : EntityBase
+    {
+        public string? Prescription { get; set; }
+
```
Comment (AzamatG17) [inline BioMed.Api/BioMed.Domain/Entities/Treatment.cs:6]: Why is there a spacebar here

**Review by AzamatG17 — APPROVED** [review]

File: BioMed.Api/BioMed.Domain/Entities/Treatment.cs (hunk 2 above)
Comment (khojianvar) [reply BioMed.Api/BioMed.Domain/Entities/Treatment.cs:6]: To separate property with navigation property.

**Review by Zikirillo2002 — APPROVED** [review]
All Entities are created without defects.

File: BioMed.Api/BioMed.Domain/Entities/Spesialization.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿namespace BioMed.Domain.Entities
+{
+    public class Spesialization : EntityBase
+    {
+        public string? Name { get; set; }
+
```
Comment (AzamatG17) [inline BioMed.Api/BioMed.Domain/Entities/Spesialization.cs:6]: You didn't add the doctor here, I corrected it

**Pull request merged**